web/html/src/components/table/SearchField.tsx

```diff
@@ -1,17 +1,30 @@
 import * as React from "react";
 
+import ReactSelect from "react-select";
+
 type SearchFieldProps = {
-  /** Search criteria value */
+  /** The value that the user will enter to perform the filter on the criteriaField */
   criteria?: string;
 
+  /** The field on which the user like to perform the filter */
+  field?: string | any;
```

Comment:
The type `string | any` will collapse into just `any`, is there a better type we could use here?

Reply:
I believe this is a mistake I forgot to fix, should be `string` only